public/pages/cadastro/cadastro_pais.html

```diff
@@ -23,7 +23,7 @@
     <header class="container-fluid p-0 navbar-container">
       <nav class="navbar navbar-expand-lg">
         <a class="navbar-brand .visible-brand-inMobile" href="/index.html">
-          <img class="img-brand" src="/public/assets/images/logo.png">
+          <img class="img-brand" src="./public/assets/images/logo.png">
```

Comment:
Sugestão de correção:

```suggestion
          <img class="img-brand" src="../../../assets/images/logo.png">
```